tests/rest/admin/test_room.py

```diff
@@ -1792,6 +1795,134 @@ def test_get_joined_members_after_leave_room(self) -> None:
         self.assertEqual(403, channel.code, msg=channel.json_body)
         self.assertEqual(Codes.FORBIDDEN, channel.json_body["errcode"])
 
+    def test_topo_token_is_accepted(self) -> None:
+        user = self.register_user("foo1", "pass")
+        user_tok = self.login("foo1", "pass")
+        room_id = self.helper.create_room_as(user, tok=user_tok)
+
+        token = "t1-0_0_0_0_0_0_0_0_0"
+        channel = self.make_request(
+            "GET",
+            "/_synapse/admin/v1/rooms/%s/messages?access_token=%s&from=%s"
+            % (room_id, self.admin_user_tok, token),
+        )
+        self.assertEqual(HTTPStatus.OK, channel.code)
```

Comment:
```suggestion
        self.assertEqual(200, channel.code)
```

There was a discussion and decision by Synapse Team to use in tests integers.